src/FAudio_platform_sdl2.c

```diff
@@ -26,6 +26,7 @@
 
 #include "FAudio_internal.h"
 
+#ifndef FAUDIO_WIN32_PLATFORM
```

Comment:
Can go above FAudio_internal.h, to avoid preprocessing extra stuff (may need an `extern int this_tu_is_empty` like we do in other files)